cpu-monitoring-example/package.json

```diff
@@ -4,26 +4,44 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
-    "build": "tsc -b && vite build",
-    "lint": "eslint .",
-    "preview": "vite preview"
+    "build": "vite build",
+    "prebuild": "npm run i18n:compile",
+    "watch": "vite build --watch",
+    "prewatch": "npm run i18n:compile",
+    "serve": "vite preview",
+    "i18n:extract": "formatjs extract 'src/**/*.ts*' --ignore='**/*.d.ts' --out-file src/i18n/langs/en.json --id-interpolation-pattern '[sha512:contenthash:base64:6]'",
+    "i18n:compile": "formatjs compile-folder src/i18n/langs --ast src/i18n/langs-compiled",
+    "format": "prettier --write .",
+    "check-types": "tsc"
   },
   "dependencies": {
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0"
```

Comment:
Moving from react 19 to 18.3 is a major downgrade. While this could work, i suggest to use the latest version